packages/in_app_purchase/lib/src/store_kit_wrappers/sk_product_wrapper.dart

```diff
@@ -0,0 +1,255 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:flutter/services.dart';
+import 'package:flutter/foundation.dart';
+import 'package:in_app_purchase/src/channel.dart';
+
+/// A product request.
+///
+/// This is a wrapper for StoreKit's
+/// [SKProductsRequest](https://developer.apple.com/documentation/storekit/skproductsrequest?language=objc).
+class SKProductRequestWrapper {
```

Comment:
@amirh I am naming this to SKProductRequestWrapper since it is resemble the underlining objc code as well. (Without the cancel method for now)